.github/workflows/push.yaml

```diff
@@ -140,7 +140,11 @@ jobs:
           linting: on
           commit_username: stakater-user
           commit_email: stakater@gmail.com
-          
+
+      # Generate manifest release
+      - name: Generate manifest
+        run: helm template forecastle deployments/kubernetes/chart/forecastle/ --include-crds > deployments/kubernetes/forecastle.yaml
+
```

Comment:
can you please add this command as well 

`helm template forecastle deployments/kubernetes/chart/forecastle/ --include-crds --output-dir deployments/kubernetes/manifests/ && mv deployments/kubernetes/manifests/forecastle/templates/* deployments/kubernetes/manifests/ && mv deployments/kubernetes/manifests/forecastle/crds/* deployments/kubernetes/manifests/ && rm -r deployments/kubernetes/manifests/forecastle`

this will populate update manifest folder with latest version of manifests